modules/virt-vm-fields-web.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+// VM wizard includes additional options to VM template wizard
+// Call appropriate attribute in the assembly
+
+[id="virt-vm-fields-web_{context}"]
+= Virtual machine web console fields
+
+The following table outlines the editable fields when creating a new virtual machine (VM) in the {VirtProductName} web console.
+
+|===
+|Name |Description
+
+|Overview
+a|
+* Description
+* Operating System
+* CPU/Memory
+* Boot Mode
+* GPU devices
+* Host Devices
```

Comment:
Capitalization needs to be checked against the [UI](https://github.com/kubevirt-ui/kubevirt-plugin/pull/403). Unfortunately, the UXD team is not consistent. They are supposed to use sentence case. I'll open an issue.

Reply:
Pretty sure the weird capitalization was me and not the UI. Fixed in doc.